server/routes/functionalSkills/functionalSkillsController.test.ts

```diff
@@ -68,8 +73,8 @@ describe('functionalSkillsController', () => {
 
       prisonService.lookupPrison.mockImplementation(mockPrisonLookup)
 
-      const functionalSkills: FunctionalSkills = {
-        problemRetrievingData: false,
+      const functionalSkills = validFunctionalSkills({
```

Comment:
Update some tests to use the new test data builders that I had to create as part of this work